ui/jstests/test-learning-resource.jsx

```diff
@@ -554,8 +561,60 @@ define(['QUnit', 'jquery', 'react', 'lodash', 'learning_resources',
     function(assert) {
       var div = document.createElement("div");
       assert.ok($(div).html().length === 0);
-      LearningResources.loader("repo", "1", function() {}, function() {}, div);
+      LearningResources.loader(
+        "repo", "1", function() {}, function() {}, function() {}, false,
+        false, div
+      );
       assert.ok($(div).html().length > 0);
     }
   );
+
+  QUnit.test('Assert learning resource panel does close',
+    function (assert) {
+      var done = assert.async();
+      var lrMountCount = 0;
+      var closeLearningResourcePanelCount = 0;
+      var closeLearningResourcePanel = function() {
+        closeLearningResourcePanelCount += 1;
+      };
+
+      var Container = React.createClass({
+        getInitialState: function() {
+          return {closeLRPanel: false};
+        },
+        render: function() {
+          lrMountCount += 1;
```

Comment:
We can't rely on counting how many times this function is called. What is the actual condition that you are waiting for?